Blockzilla/OverlayView.swift

```diff
@@ -15,19 +15,40 @@ protocol OverlayViewDelegate: class {
 
 class OverlayView: UIView {
     weak var delegate: OverlayViewDelegate?
-    private var searchButtonGroup = [InsetButton]()
-    private let searchSuggestionsCount = 4 // Five search buttons in total since indexing starts from 0.
-    private var presented = false
-    private var searchQuery = ""
     private let copyButton = UIButton()
     private let findInPageButton = InsetButton()
+    private let searchSuggestionsPrompt = SearchSuggestionsPromptView()
+    private let searchButton = InsetButton()
```

Comment:
I'm a little confused as to what's going on with ```searchButton`` and ```searchButtonGroup[0]```. Is there a difference? It's sort of confusing from a readability perspective.

Reply:
Woops, searchButton is an artefact from a previous commit, removing it!